src/Ledger/Conway/Specification/Transaction.lagda.md

```diff
@@ -223,17 +244,14 @@ record TransactionStructure : Type₁ where
 
     HasDonations-Tx : HasDonations Tx
     HasDonations-Tx .DonationsOf = DonationsOf ∘ TxBodyOf
-\end{code}
-\end{NoConway}
-\end{AgdaMultiCode}
-\caption{Transactions and related types}
-\label{fig:defs:transactions}
-\end{figure*}
-
-\begin{NoConway}
-\begin{figure*}[ht]
-\begin{AgdaMultiCode}
-\begin{code}
+```
+-->
+
+
+## Transaction Functions
```

Comment:
Auxiliary functions?

Reply:
I think it's better if the header of the section of each module where functions related to that module are defined should refer to the name of the module itself; we already do this elsewhere, e.g., "Certification Functions" (or "Certificates Functions"?), "Governance Functions", etc.  Making the subsection titles more specific, less generic, seems preferable to me.